### DISCUSSION TITLE
Wix Bundle and Install time settings

### DISCUSSION THREAD

**Discussion by syedharoonalam** [wixtoolset/issues#6479] [Questions]
I am creating a Wix installer. My app needs a prereq of edge runtime So I install edge runtime and then MSI using Bootstrapper/Bundle

Next, I need to introduce per user / per machine options for the user to select in my MSI. The problem here is that bootstrapper doesn't allow to have install time settings.

(For example, in my case if I set bundle per machine, and then at install time use per user for MSI, at the time of uninstalling it skips registry values in per user (which makes sense as the bundle was set to per machine), a Hack could be to call uninstaller of MSI rather than bundle - but this may not be the best way to do it I think)

What would be the right way to solve this problem?

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
